.github/workflows/release-binary.yml

```diff
@@ -2,18 +2,21 @@ name: Release Binaries
 
 on:
   release:
-    types: [ published ]
+    types: [published]
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
 jobs:
   binary:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Are we sure this is correct? Go releaser will run twice the job on both arch.
I don't think it makes sense here.